packages/xod-project/src/node.js

```diff
@@ -160,6 +161,27 @@ export const getNodePosition = def(
   R.prop('position')
 );
 
+/**
+ * @function setNodeArityLevel
+ * @param {ArityLevel} arityLevel - new arity of the node
+ * @param {Node} node - node to update
+ * @returns {Node} copy of node with new arity level
+ */
+export const setNodeArityLevel = def(
+  'setNodeArityLevel :: ArityLevel -> Node -> Node',
+  R.assoc('arityLevel')
+);
```

Comment:
Can it be so simple for public API? If one sets the level to 10, link all the pins, and then sets level back to 1, who is responsible for dropping the links?